confgenerator/agentmetrics.go

```diff
@@ -41,6 +41,10 @@ func (r MetricsReceiverAgent) Pipeline() otel.Pipeline {
 			},
 		},
 		Processors: []otel.Component{
+			{
```

Comment:
Please split this change (I guess including the submodule update and label change) into a separate PR so we can review and test it separately, since it changes so many files.

But also... is there something different that the new code in the agentmetrics processor is doing that's different from the `otel.AggregateLabels` calls you added in `hostmetrics.go`? Can it be handled with a smaller diff by just adding calls to `otel.AggregateLabels` in the `MetricsTransform` below?

Reply:
Sorry, I missed that this was still in there. The AggregateLabels calls should actually be irrelevant, but I believe the agentmetrics processor will still be necessary for resolving the issue that we attempted to resolve back in https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/pull/59#discussion_r733070787 the 0.37 update PR. 

There were two different issues with ops agent integration tests that were encountered during this PR, the swap/bytes_used that I was attempting to resolve with AggregateLabels, and the service_version label issue that was being resolved with the agentmetrics processor added to the agentmetrics pipeline.

Reply:
https://github.com/GoogleCloudPlatform/ops-agent/pull/261

Reply:
#261 is merged